docs/requirement-sets/word/word-api-desktop-requirement-set.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Word JavaScript API desktop-only requirement set
+description: Details about the WordApiHiddenDocument requirement set.
+ms.date: 06/24/2022
+ms.prod: word
+ms.localizationpriority: medium
+---
+
+# Word JavaScript API desktop-only requirement set
+
+The `WordApiHiddenDocument` requirement set is a special requirement set that includes features that are only available for Word on Windows and on Mac. APIs in this requirement set are considered to be production APIs for the Word application on Windows and on Mac. They follow [Microsoft 365 developer support policies](/office/dev/add-ins/publish/maintain-breaking-changes). `WordApiHiddenDocument` APIs are considered to be "preview" APIs for other platforms (web, iPad) and may not be supported by any of those platforms.
+
+When APIs in the `WordApiHiddenDocument` requirement set are supported across all platforms, they will added to the next released requirement set (`WordApi 1.[NEXT]`). Once that new requirement is public, those APIs will be removed from `WordApiHiddenDocument`. Think of this as a similar promotion process to an API moving from preview to release.
+
+> [!IMPORTANT]
+>
+> - `WordApiHiddenDocument` is a superset of the latest numbered requirement set.
+> - `WordApiHiddenDocument 1.3` is currently the only version of the desktop-only APIs.
+
+## Recommended usage
+
+Because `WordApiHiddenDocument` APIs are only supported by Word on Windows and on Mac, your add-in should check if the requirement set is supported before calling these APIs. This avoids any attempt to use desktop-only APIs on an unsupported platform.
+
+```js
+if (Office.context.requirements.isSetSupported("WordApiHiddenDocument", "1.3")) {
+   // Any API exclusive to the desktop-only requirement set.
+}
+```
+
+Once the API is in a cross-platform requirement set, you should remove or edit the `isSetSupported` check. This will enable your add-in's feature on other platforms. Be sure to test the feature on those platforms when making this change.
+
+> [!IMPORTANT]
+> Your manifest cannot specify `WordApiHiddenDocument 1.3` as an activation requirement. It's not a valid value to use in the [Set element](/javascript/api/manifest/set).
```

Comment:
Should we document this in the Set manifest page?

Reply:
I hesitate to note this as the only exception. I know there are other exceptions so perhaps we can throw this on the backlog?

Reply:
[Internal item](https://office.visualstudio.com/OC/_workitems/edit/6183397)

Reply:
Fixed in [PR 1308](https://github.com/OfficeDev/office-js-docs-reference/pull/1308)